integration/loki_micro_services_test.go

```diff
@@ -710,6 +711,272 @@ func TestQueryTSDB_WithCachedPostings(t *testing.T) {
 
 }
 
+func TestCategorizedLabels(t *testing.T) {
+	clu := cluster.New(nil, cluster.SchemaWithTDSBAndNonIndexedLabels)
```

Comment:
let's rename it to `SchemaWithTDSBAndStructuredMetadata`

Reply:
Done